### PR TITLE
add value for ie11 to avoid the double calc

### PR DESCRIPTION
**Pull request by pochretien**
fix the body size in ie11

### CODE REVIEW

**Comment by pochretien** [comment]
image

image

File: scss/components/prompt.scss [hunk 1]
```diff
@@ -26,9 +26,10 @@ $prompt-message-line-height: 20px;
 
       .modal-body {
         // Reset flex-basis and flex-grow values so autosize based on content can work.
-        flex-basis: initial;
+        flex-basis: inherit;
```
Comment (JPLachance) [inline scss/components/prompt.scss]: Hum, what does that change?

Comment (pochretien) [inline scss/components/prompt.scss]: For ie11, the modal-body can now take the size of his parent. This change avoid the content to always take the min-height. Nothing changes for other browsers.

Comment (JPLachance) [inline scss/components/prompt.scss]: okay!

**Pull request merged**